token/services/db/sql/common/tcondition.go

```diff
@@ -59,6 +67,15 @@ func (c *tokenInterpreter) HasTokenDetails(params driver.QueryTokenDetailsParams
 	if !params.IncludeDeleted {
 		conds = append(conds, common.ConstCondition("is_deleted = false"))
 	}
+	if params.OnlyNonSpendable {
```

Comment:
This allows us to set both properties with unexpected results. We can have an enum property `Spendable` with `TRUE`, `FALSE`, `ANY`.